cpp/src/arrow/util/compression.h

```diff
@@ -33,6 +33,28 @@ struct Compression {
 
 namespace util {
 
+inline int32_t GetCompressionCodecDefaultCompressionLevel(Compression::type t) {
```

Comment:
There doesn't seem to be any point in exposing this function in a header file, is there?

Reply:
Thanks, I moved it to the source file.